docs/08-models-and-migrations.md

```diff
@@ -428,7 +477,28 @@ Of course, we want to prove that any model using our `HasPost` trait can indeed
 
 Therefore, we’ll create a new `User` model, not within the `src/Models/` directory, but rather in our `tests/` directory.
 
-In the `User` model we’ll use the same traits that would be available on the `User` model that ships with a standard Laravel project to stay close to a real world scenario. Also, we use our own `HasPosts` trait:
+To be able to create users within our tests we'll need to overwrite the `UserFactory` provided by the Orchestra Testbench package, as shown bellow.
+
+```php
+// 'tests/UserFactory.php'
+<?php
+
+namespace JohnDoe\BlogPackage\Tests;
+
+use Orchestra\Testbench\Factories\UserFactory as FactoriesUserFactory;
+
+class UserFactory extends FactoriesUserFactory
+{
+  /**
+   * The name of the factory's corresponding model.
+   *
+   * @var string
+   */
+  protected $model = User::class;
+}
```

Comment:
I suppose we also should define a definition for this factory, right?

```suggestion

    /**
     * Define the model's default state.
     *
     * @return array
     */
    public function definition()
    {
        return [
            'name' => $this->faker->name,
            'email' => $this->faker->unique()->safeEmail,
            'email_verified_at' => now(),
            'password' => bcrypt('password'),
            'remember_token' => \Illuminate\Support\Str::random(10),
        ];
    }
}
```

Reply:
Right!